pkgs/by-name/ch/chainsaw/package.nix

```diff
@@ -8,20 +8,18 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "chainsaw";
-  version = "2.9.2";
+  version = "2.10.1";
 
   src = fetchFromGitHub {
     owner = "WithSecureLabs";
     repo = "chainsaw";
     rev = "refs/tags/v${version}";
-    hash = "sha256-oKlLFKCZNBcHURVvT0i8Hfym6r30ikndV30uuxjYgDM=";
+    hash = "sha256-ErDIfLhzCiFm3dZzr6ThjYCplfDKbALAqcu8c0gREH4=";
   };
 
-  cargoHash = "sha256-1w3DYP69aOphoqbsZIF9aX56Lftg2niKClmImLTa5DE=";
+  cargoHash = "sha256-WuPfYxI61wsQyOKqPF7/a3Hx9vMBCkTkoXsZczgeKpg=";
 
-  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [
-    darwin.apple_sdk.frameworks.CoreFoundation
-  ];
+  buildInputs = lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.CoreFoundation ];
```

Comment:
hm, that's interesting, as the only thing I've done after your last comment was running `nixfmt` from `nixfmt-unstable` 

should I change it to `stdenv.hostPlatform.isDarwin` by hand then?

Reply:
Weird, let's revert it to `stdenv.hostPlatform.isDarwin` since this was not intentional.

Reply:
seems like the check for formatting is failing then 🤔

Reply:
Re-run `nixfmt-rfc-style`.
The `buildInputs` line should look like so:
```nix
  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [
    darwin.apple_sdk.frameworks.CoreFoundation
  ];
```

Reply:
Looks like a new line has been inserted at the EOF.

Reply:
okay I have no ideas why it fails, I've literally deleted the file locally, recreated it, re-run `nixfmt` on it and the check for formatting is still failing, and the only one that has been successful was the one with `stdenv.isDarwin` instead of `stdenv.hostPlatform.isDarwin` for some reason

Reply:
This fail again because of the EOF, which you can confirm by checking the details of the failed checks.

Is your editor automatically does some modification to the end of file or something?

Reply:
![image](https://github.com/user-attachments/assets/cf22a596-80ff-417f-a1a7-cd853fb261f5)

doesn't look like it...? 
can I just revert to version which passed that check? (that is - `stdenv.isDarwin` one)

Reply:
`git apply` this patch and it should be good.
```diff
diff --git a/pkgs/by-name/ch/chainsaw/package.nix b/pkgs/by-name/ch/chainsaw/package.nix
index 1af5a5fc7..f345d897c 100644
--- a/pkgs/by-name/ch/chainsaw/package.nix
+++ b/pkgs/by-name/ch/chainsaw/package.nix
@@ -36,4 +36,4 @@ rustPlatform.buildRustPackage rec {
     maintainers = with maintainers; [ fab ];
     mainProgram = "chainsaw";
   };
-}
\ No newline at end of file
+}
```

Reply:
seems like the formatting check has passed

thx for all the help, let's just hope the rest of the checks pass 

I will investigate further later on what's the cause of the issue I'm having, my wild guess would be the fact that this was done on WSL, and I've cloned my fork of nixpkgs on Windows part of the filesystem, so I suspect that might have something to do with it, but, as said, that's just my wild guess